docs/commands/graph.query.md

```diff
@@ -4,16 +4,35 @@ Arguments: `Graph name, Query, Timeout [optional]`
 
 Returns: [Result set](/redisgraph/design/result_structure)
 
+### Queries and Parameterized Queries
+
+The execution plans of queries, both regular and parameterized, are cached (up to [CACHE_SIZE](https://redis.io/docs/stack/graph/configuration/#cache_size) execution plans are cached). Therefore, it is recommended to use parametrized queries when using many queries with the same pattern but different parameters.
```

Comment:
Therefore, it is recommended to use parametrized queries when executing many queries with the same pattern but different constants.